server/backend/quiz/models.py

```
@@ -0,0 +1,51 @@
+from django.conf import settings
+
+class Category(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.name
+
+    @property
+    def question_count(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `question_count`

(ANN201)